be/test/exprs/string_fn_test.cpp

```diff
@@ -622,6 +622,126 @@ PARALLEL_TEST(VecStringFunctionsTest, splitConst2) {
     ASSERT_TRUE(StringFunctions::split_close(ctx.get(), FunctionContext::FunctionStateScope::FRAGMENT_LOCAL).ok());
 }
 
+TypeDescriptor array_type(const LogicalType& child_type);
```

Comment:
How about writing test cases in sql-test that is more convenient?

Reply:
some cases should be written in UT, for example constructing const columns, only-null columns.